tests/fuzz/s2n_kyber_r3_recv_public_key_fuzz_test.c

```diff
@@ -13,7 +13,7 @@
  * permissions and limitations under the License.
  */
 
-/* Target Functions: s2n_kem_recv_public_key s2n_kem_encapsulate kyber_512_r3_crypto_kem_enc */
+/* Target Functions: s2n_kem_recv_public_key s2n_kem_encapsulate kyber_512_r3_crypto_kem_enc kyber_768_r3_crypto_kem_enc kyber_1024_r3_crypto_kem_enc */
```

Comment:
Do we really want to maintain this list? Is there any guarantee that we update it when we update the implementation?

Reply:
This appears to be the [recommended way](https://github.com/aws/s2n-tls/blob/05ae36ce470f20233821f3cb000140124023b6bb/tests/fuzz/Readme.md?plain=1#L18) of designating fuzz targets. Similar comments can be found [throughout the codebase](https://github.com/search?q=repo%3Aaws%2Fs2n-tls+%22Target+Functions%22&type=code). Perhaps there's room to automate that by parsing header files or something, but I think that redesign would be out of scope for this change.